roles/inspec/defaults/main.yml

```diff
@@ -24,10 +24,10 @@ inspec:
     rhel7:
       enabled: true
       profile: stig_rhel
-      skip_controls: []
+      skip_controls: [RHEL-07-010010,RHEL-07-010020,RHEL-07-010371,RHEL-07-010372,RHEL-07-010373,RHEL-07-010072,RHEL-07-040810,RHEL-07-040820,RHEL-07-020140,RHEL-07-010500,RHEL-07-020130,RHEL-07-030810,RHEL-07-030820,RHEL-07-040290,RHEL-07-040250,RHEL-07-040480,RHEL-07-040010,RHEL-07-040300,RHEL-07-030524,RHEL-07-030525,RHEL-07-030526,RHEL-07-030530,RHEL-07-030531,RHEL-07-030540,RHEL-07-030541,RHEL-07-030550,RHEL-07-030560,RHEL-07-030561,RHEL-07-030630,RHEL-07-030670,RHEL-07-030671,RHEL-07-030672,RHEL-07-030673,RHEL-07-030310,RHEL-07-030674,RHEL-07-030750,RHEL-07-030751,RHEL-07-030752,RHEL-07-030753,RHEL-07-030754,RHEL-07-030380,RHEL-07-030381,RHEL-07-030382,RHEL-07-030383,RHEL-07-030390,RHEL-07-030391,RHEL-07-030392,RHEL-07-030400,RHEL-07-030401,RHEL-07-030402,RHEL-07-030403,RHEL-07-030404,RHEL-07-030405,RHEL-07-030420,RHEL-07-030421,RHEL-07-030422,RHEL-07-030423,RHEL-07-030424,RHEL-07-030425,RHEL-07-030441,RHEL-07-030442,RHEL-07-030443,RHEL-07-030444,RHEL-07-020160,RHEL-07-020170,RHEL-07-020200,RHEL-07-020211,RHEL-07-020650,RHEL-07-020700,RHEL-07-020860,RHEL-07-021160,RHEL-07-021230,RHEL-07-021240,RHEL-07-021250,RHEL-07-021270,RHEL-07-021280,RHEL-07-010280,RHEL-07-010380,RHEL-07-010420,RHEL-07-010460,RHEL-07-030510,RHEL-07-030511,RHEL-07-030512,RHEL-07-030513,RHEL-07-030514,RHEL-07-030521,RHEL-07-030522,RHEL-07-021260,RHEL-07-030090,RHEL-07-030330,RHEL-07-030331,RHEL-07-030340,RHEL-07-030350,RHEL-07-030351,RHEL-07-030490,RHEL-07-030491,RHEL-07-030492,RHEL-07-030523,RHEL-07-010240,RHEL-07-040160,RHEL-07-030770,RHEL-07-040020,RHEL-07-010130,RHEL-07-010140,RHEL-07-010150,RHEL-07-010160]
       attributes:
         RHEL_07_030710_audit_key_name: audit_account_changes-V-38531
-        client_alive_interval: "{{ common.ssh.client_alive_interval }}"
+        client_alive_interval: "'{{ common.ssh.client_alive_interval }}'"
```

Comment:
ugh I'd rather not do this ...  the inspec check should be smart enough to not need to have hard coded quote marks as they could be ' or ".    I'd suggest making a PR to the inspec itself.